docs/helm/endpoints_api.md

```diff
@@ -13,6 +13,7 @@
 *  **URL Params**
 
    `ns=[string]` - Namespace
+   `limitInfo=[boolean]` - limitInfo
```

Comment:
@Kartikey-star @debsmita1 I think you both were correct in your discussion about this parameter. Aka, there is no correct or incorrect, but I can want to add my opinion here as well... :laughing:

This is just an opinion that's the reason I'm **_fine_** with merging your change, but this "new limited API" should be the **_default_**.

With that in mind, I would add the parameter for "the special case" topology, but Debsmita is right that `isTopology` is then again too specific... This is how I would name it:

```suggestion
includeManifest=[boolean]` - Optional and deprecated parameter to include the full Helm manifest for all releases.
```

But you know, [naming things is hard](https://martinfowler.com/bliki/TwoHardThings.html), and we can go with limitInfo.

I hope we can have a follow-up PR that updates the UI so that we can **_remove_** `limitInfo` completely, always return the smaller data set, and then lazy load the Helm manifest/resources in the topology only when the sidebar is opened. From my understanding that's the reason, we need this info there. And this would be a big improvement for the topology.

Reply:
Created https://issues.redhat.com/browse/ODC-7227 to cater to above mentioned comments. Thanks @jerolimov .